pyaedt/icepak.py

```diff
@@ -684,7 +684,7 @@ def create_source_power(
         if input_power == 0:
             input_power = "0W"
         if not bool(input_power) ^ bool(thermal_dependent_dataset):
-            self.logger.error("Please assigned one input between ``thermal_dependent_dataset`` and  ``input_power``")
+            self.logger.error("Assign one input between ``thermal_dependent_dataset`` and  ``input_power``")
```

Comment:
@lorenzovecchietti Thanks! I would like to see a period added before the closing double quote though!

Reply:
```suggestion
            self.logger.error("Assign one input between ``thermal_dependent_dataset`` and  ``input_power``.")
```